src/content/app/entity-viewer/state/gene-view/transcripts/geneViewTranscriptsSlice.ts

```diff
@@ -14,9 +14,13 @@
  * limitations under the License.
  */
 
-import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
-import { Action } from 'redux';
-import { ThunkAction } from 'redux-thunk';
+import {
+  createAsyncThunk,
+  createSlice,
+  type PayloadAction
+} from '@reduxjs/toolkit';
+import type { Action } from 'redux';
+import type { ThunkAction } from 'redux-thunk';
```

Comment:
Here and elsewhere: type Action and type ThunkAction are imported by redux-toolkit and then re-exported ([link1](https://github.com/reduxjs/redux-toolkit/blob/master/packages/toolkit/src/index.ts#L2), [link2](https://github.com/reduxjs/redux-toolkit/blob/master/packages/toolkit/src/index.ts#L19). Could you import them from `@reduxjs/toolkit`?

Reply:
That's done with the latest commit [5a12a55](https://github.com/Ensembl/ensembl-client/pull/978/commits/5a12a55d6f6ed514bd216b55a348b8624087ad20). Also, fixed a few other type imports as well.

✅